readme.md

```diff
@@ -1,4 +1,5 @@
-#Yahoo Answers Scraper
+## !!! Important: The Yahoo Answers API is now deprecated and this code no longer functions !!!
```

Comment:
Can you remove the !!!'s and replace "deprecated" with "removed". I usually think deprecated means still supported, but not for long.

Also, can you add the link to the Yahoo announcement: https://yahoodevelopers.tumblr.com/post/86260183503/yahoo-answers-apis-will-be-removed-as-of-june-3